mgmt/rest/v2/plugin.go

```diff
@@ -148,11 +175,12 @@ func (s *apiV2) loadPlugin(w http.ResponseWriter, r *http.Request, _ httprouter.
 		return
 	}
 
+	var certPath, keyPath, caCertPaths string
 	if strings.HasPrefix(mediaType, "multipart/") {
```

Comment:
Declaration of vars should be moved inside this `if` statement. There is no use of those vars outside.